### PR TITLE
R4 Final Review

### PR DESCRIPTION
**Pull request by amsmith-pro**
No major changes, just added a hardcoded set of waypoints and zones. The hardcoded waypoints will be removed in R5 after the automatic waypoint generation feature is merged in.

### CODE REVIEW

**Review by owlot — CHANGES_REQUESTED** [review]
Rust checks are failing

**Comment by owlot** [comment]
## Documents
- [x] ICD
  - [x] Current interfaces (grpc & rest)
  - [x] Current relevant document links
- [x] SDD
  - [x] Description of grpc & rest handler implementations
  - [x] Current sequence diagrams where applicable
  - [x] Current relevant document links
- [x] CONOPS
  - [x] Current information
  - [x] Bibliography (if any citations)
- [x] README (see svc-cargo for example)
  - [x] Badges
  - [x] Accurate installation instructions
  - [x] Current links to other documents
- [x] Any instance of :construction: should genuinely be under construction

## Code (not applicable to examples/ folder)
- [x] No unhandled unwrap()s
- [x] Accurate (!) code comments on all functions
- [x] No `# Arguments` and `# Returns` in function comments
- [x] Function arguments checked where possible
- [x] Sufficient logger (info, warn, and debug) messages
- [x] Sensible Types (unsigned vs. signed)
- [x] No `#[allow(dead_code)]` without good reason
- [x] End all error, info, and debug messages with punctuation (periods, etc.)
- [x] No FIXMEs or TODOs without a projected release (`TODO R4: etc.`)

## REST API
- [x] Return codes accurately documented with openapi
- [x] Return codes make sense
- [x] Sufficiently descriptive error messages
- [x] Do NOT reflect a user's bad input back at them in replies.
    - i.e. "Warning: '{}' is not a valid entry." should instead be "Warning: invalid entry."
    - Use debug messages to see the invalid argument.

## Cargo.toml(s)
- [x] Package versions correct
- [x] No minor versions specified
- [x] Links to crates.io package, not git branch
- [x] Must have `license-file = "LICENSE"` under `[package]`

File: LICENSE [hunk 1]
```diff
@@ -7,17 +7,17 @@ License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
...
+                      The Licensed Work is (c) 2024 Aetheric B.V.
 
 Additional Use Grant: Limited production uses, or as otherwise permitted by
-                      Arrow DAO governance vote
+                      Aetheric B.V. governance vote
 
 Change Date:          for each release of svc-compliance, the earlier of (i) two
                       years from the release date, or (ii) a date specified at
-                      https://github.com/Arrow-air/svc-compliance/blob/main/README.md
+                      https://github.com/aetheric-sso/svc-compliance/blob/main/README.md
```
Comment (owlot) [inline LICENSE]: Seems like this branch needs a rebase from develop, `sso` vs `oss` typo is still present here.

File: server/src/region/us.rs
Comment (owlot) [inline server/src/region/us.rs:1]: Log message function name prefixes should be removed

File: LICENSE (hunk 1 above)
Comment (amsmith-pro) [reply LICENSE]: pulled in develop

File: server/src/region/us.rs
Comment (amsmith-pro) [reply server/src/region/us.rs:1]: https://github.com/aetheric-oss/svc-compliance/commit/5beaa1a89072f9c5e3488e04c5f785cf12b65cf1

**Review by owlot — APPROVED** [review]

**Pull request merged**